notifier.go

```diff
@@ -17,7 +17,7 @@ import (
 )
 
 const notifierName = "gobrake"
-const notifierVersion = "5.0.3"
+const notifierVersion = "5.1.0"
```

Comment:
Probably we need to release v5.1.1 so that this constant is updated. You can do it in this same PR. Please make sure to update changelog as well.